server/models/MealItem.py

```diff
@@ -0,0 +1,35 @@
+from . import db, bcrypt
+from marshmallow import Schema
+
+
+class MealItem(db.Model):
+    __tablename__ = 'meal_items'
+
+    id = db.Column(db.Integer, primary_key=True)
+    userId = db.Column(
+        db.Integer,
+        db.ForeignKey('users.id', ondelete='CASCADE'),
```

Comment:
Just want to double check - this ondelete="CASCADE" means when the user is deleted, we remove the meal items right?

Reply:
Yes